DOC-7519 -- Pending Doc Ids not documented

ibsoln opened the pull request:
DOC-7519 -- Pending Doc Ids not documented

https://issues.couchbase.com/browse/DOC-7519

ibsoln commented:
Can you review the code in the below commits please? I added a snippet to show PendindDocIDs and IsDocumentPending in action.

- @bmeike  -- JAVA Snippets … 3a57b76
- @jayahariv   -- iOS Snippets …425fabb
- @Sandychuang8  -- C# Snippets … 400daa5

jayahariv commented on modules/objc/examples/code_snippets/SampleCodeTest.m:
```diff
@@ -836,6 +836,66 @@ - (void) dontTestReplicatorStatus {
+          NSLog(@"Replicator activity level is %u", change.status.activity);
+          // iterate and report-on the pending doc IDs  in 'mydocids'
+
+          NSEnumerator *enumerator =
+            [mydocids objectEnumerator];
+          id thisid;
+
+          while (thisid =[ enumerator nextObject]) {
```
probably more easier?
```
for (thisid in mydocids) {
  NSError* err = nil;
  if (![replicator isDocumentPending: thisid error: &err]) { // <.>
    NSLog(@"Doc ID %@ now pushed", thisid);
  }
}

ibsoln replied:
Thanks @jayahariv

jayahariv reviewed (approved):
looks ok

ibsoln committed:
DOC-7519 -- Pending Doc Ids not documented

https://issues.couchbase.com/browse/DOC-7519

merged prior commits ...

DOC-7519 -- JAVA Snippets

https://issues.couchbase.com/browse/DOC-7519

Pending Doc Ids not documented

DOC-7519 -- iOS Snippets

https://issues.couchbase.com/browse/DOC-7519

Pending Doc Ids not documented

DOC-7519 -- C# Snippets

https://issues.couchbase.com/browse/DOC-7519

Pending Doc Ids not documented

DOC-7519 -- Pending Doc Ids not documented

https://issues.couchbase.com/browse/DOC-7519

DOC-7519 -- iOS Snippets (objc)
Simplify iteration look at ln 875
https://issues.couchbase.com/browse/DOC-7519

Revert "DOC-7519 -- iOS Snippets (objc)"

This reverts commit d9ee906c646622db95204e282bbd4aa5b265e1aa.

DOC-7519 -- iOS Snippets (objc)

https://issues.couchbase.com/browse/DOC-7519

Simplify iteration look at ln 875

DOC-7519 - Apply changes to Java snippets
https://issues.couchbase.com/browse/DOC-7519
